lib/node-token-validation/src/api/TokenValidator.ts

```diff
@@ -285,8 +542,10 @@ export class TokenValidator {
 
     /**
      * Function to check that validAudiences parameter is not an empty array. Throws emptyAudience error if empty.
-     * @param {@link (BaseValidationParameters:type)} 
-     * @returns 
+     * 
+     * @param {BaseValidationParameters} options Built validation parameters
+     * @returns {string[]} Returns an array of valid audiences
+     * @ignore
```

Comment:
What does ignore do in this context?

Reply:
The @ignore in this context means this function will not be documented in the Typedocs. I've used ignore to hide the internal helper functions, so that only the public APIs are documented in Typedocs.